Documentation/blobIn.md

```diff
@@ -1,3 +1,12 @@
+#### Connecting to your Storage Account
+
+* Download Storage explorer from here: http://storageexplorer.com/ 
+* Connect using these credentials:
+* Account Name: `{{accountName}}`
+* Account Key: `{{accountKey}}`
```

Comment:
Github secrets, WebHook/Http security codes, etc. we do show in plain text you're right. I agree that that is a slight discrepancy. In my mind showing the full storage credentials seems worse though.